### PR TITLE
updating to minicolors version 2.2.6

### PR DESCRIPTION
**Pull request by davegudge**

### CODE REVIEW

**Comment by kostia** [comment]
Hey @davegudge, thanks for providing the new version! 😄 

Unfortunately, as is it won't work in production, because the PNG image won't be bundle in the assets. See https://github.com/kostia/jquery-minicolors-rails#testing for details how to test if everything works in production.

To make it work in production, the PNG must be Base64 encoded and inline in the stylesheets as data URL. Could you please do that? Thanks!

**Comment by davegudge** [comment]
Hey @kostia, I've tested out the changes and amended the commit ;-)

File: lib/jquery-minicolors-rails/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module JqueryMinicolorsRails
-  VERSION = '2.2.3.1'
+  VERSION = '2.2.6'
```
Comment (davegudge) [inline lib/jquery-minicolors-rails/version.rb]: @kostia Does this need a `.1` on the end? 

https://github.com/kostia/jquery-minicolors-rails#versioning

Comment (kostia) [reply lib/jquery-minicolors-rails/version.rb]: Yeah, feels better if the version history looks consistent:
bildschirmfoto 2017-09-22 um 17 38 45

**Comment by kostia** [comment]
LGTM 👍 
But please also adjust the version 😄

**Comment by davegudge** [comment]
New commit added.

**Comment by kostia** [comment]
Thanks! 😄

**Comment by kostia** [comment]
https://rubygems.org/gems/jquery-minicolors-rails/versions/2.2.6.0

**Comment by davegudge** [comment]
Thank you 👍

**Pull request merged**